client/account-recovery/reset-password/reset-password-form/index.jsx

```diff
@@ -30,15 +33,52 @@ export class ResetPasswordFormComponent extends Component {
 	};
 
 	submitForm = () => {
+		// TODO:
+		// This is going to be replaced by corresponding redux actions.
 		this.setState( { isSubmitting: true } );
 	};
 
 	onResetOptionChanged = ( event ) => {
 		this.setState( { selectedResetOption: event.currentTarget.value } );
 	};
 
-	render() {
+	getOptionDisplayStrings = ( optionName ) => {
 		const { translate } = this.props;
+
+		switch ( optionName ) {
+			case 'primary':
+				return {
+					email: translate(
+						'Email a reset link to {{strong}}your main email address{{/strong}}',
+						{ components: { strong: <strong /> } }
+					),
+					sms: translate(
+						'Send a reset code to {{strong}}your main phone{{/strong}}',
+						{ components: { strong: <strong /> } }
+					),
+				};
+			case 'secondary':
+				return {
+					email: translate(
+						'Email a reset link to {{strong}}your recovery email address{{/strong}}',
+						{ components: { strong: <strong /> } }
+					),
+					sms: translate(
+						'Send a reset code to {{strong}}your recovery phone{{/strong}}',
+						{ components: { strong: <strong /> } }
+					),
+				};
+			default:
+				return '';
```

Comment:
I made a mistake in my example code. You probably should return `{}`, `null` or `undefined` here. An empty string is falsey but I think something else might be better. Maybe even take out the `default` case.